force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +297,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        String title;
+        String author;
+        Integer pages;
```

Comment:
Technically, these instance variables should all be public. Otherwise, you should get an error when you try to deploy your code.
```suggestion
        public String title;
        public String author;
        public Integer pages;
```